js/index.js

```diff
@@ -1,24 +1,36 @@
 // ITERATION 1
-
 function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
 
-  //... your code goes here
+  const price = product.querySelector('.price span');
+  const quantity = product.querySelector('.quantity span');
```

Comment:
the quantity is not defined as a span element, but as an input.
one way to get its value could be `product.querySelector(".quantity").getElementsByTagName("input")[0].value`.

one thing to remember is that getElementsByTagName returns an array (even if there is only one element)